modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/power_of_attorney_requests_spec.rb

```diff
@@ -41,13 +43,40 @@
           PoaRequestResponses.veteran_poa_response_with_extra_day(poa_requests[2], time, time_plus_one_day),
           PoaRequestResponses.declined_poa_response(poa_requests[3], time),
           PoaRequestResponses.expired_poa_response(poa_requests[4], time)
-
         ]
       )
     end
+
+    context 'when providing a status param' do
+      let!(:declined_request) { create(:power_of_attorney_request, :with_declination) }
+      let!(:pending_request) { create(:power_of_attorney_request, skip_resolution: true) }
+
+      it 'returns the list of pending power of attorney requests' do
+        get('/accredited_representative_portal/v0/power_of_attorney_requests?status=pending')
+        parsed_response = JSON.parse(response.body)
+        expect(response).to have_http_status(:ok)
+        expect(parsed_response.length).to eq 1
+        expect(parsed_response[0]['id']).to eq pending_request.id
+      end
+
+      it 'returns the list of completed power of attorney requests' do
+        get('/accredited_representative_portal/v0/power_of_attorney_requests?status=completed')
+        parsed_response = JSON.parse(response.body)
+        expect(response).to have_http_status(:ok)
+        expect(parsed_response.length).to eq 1
+        expect(parsed_response[0]['id']).to eq declined_request.id
+      end
+
+      it 'throws an error if any other status filter provided' do
+        get('/accredited_representative_portal/v0/power_of_attorney_requests?status=delete_all')
+        expect(response).to have_http_status(:internal_server_error)
```

Comment:
should be :bad_request after we reflect the changed exception type in our exception rendering.